mdtraj/formats/prmtop.py

```diff
@@ -186,11 +189,11 @@ def load_prmtop(filename, **kwargs):
 
             # check
             res_name = raw_data["RESIDUE_LABEL"][residue_pointer_dict[index]].strip()
-            if res_name in pdb.PDBTrajectoryFile._residueNameReplacements:
+            if res_name in pdb.PDBTrajectoryFile._residueNameReplacements and standard_names:
```

Comment:
More nitpicky than anything but I would check for `standard_names` first, then do the res_name check. Probably going to be faster, esp. if the dictionary is large.